packages/@atjson/source-html/src/converter/social-embeds.ts

```diff
@@ -25,6 +25,14 @@ function isBlockquoteEmbed(annotation: Annotation<any>) {
   );
 }
 
+interface attribute {
```

Comment:
Could you please rename this interface to something more descriptive, and capitalize it?